pkg/version/version.go

```diff
@@ -47,3 +48,7 @@ func GetOpenShiftVersion() string {
 func GetOpenShiftSemverVersion() (semver.Version, error) {
 	return semver.Make(strings.TrimPrefix(GetOpenShiftVersion(), VersionPrefix))
```

Comment:
@hferentschik I have seen this but I guess I missed https://github.com/minishift/minishift/blob/master/pkg/minikube/update/update.go#L57